### PR TITLE
LPS-154718 | Refactor RemoteApps tests

### PR DESCRIPTION
**Pull request by mateusralv**
Description
see https://github.com/liferay-frontend/liferay-portal/pull/2594#issuecomment-1319701216  and 

Test Plan
- [ ] Test passes on CI

JIRA TICKET
https://issues.liferay.com/browse/LPS-164613

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase [hunk 1]
```diff
@@ -693,15 +693,11 @@ definition {
...
-				locator1 = "TextInput#ANY",
-				value1 = "vanilla counter");
-		}
-
-		task ("When attempt to save the remote app") {
-			Button.clickPublish();
+		task ("When type 'vanilla counter' in HTML Element field and attempt to save the remote app") {
+			RemoteApps.addApp(
+				entryName = "RemoteApp Test Name",
+				entryURL = "$testurl",
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase:699]: syntax ${} and camel case
${testURL}

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase [hunk 2]
```diff
@@ -720,15 +716,11 @@ definition {
...
-				locator1 = "TextInput#ANY",
-				value1 = "vanilla.counter");
-		}
-
-		task ("And When attempt to save the remote app") {
-			Button.clickPublish();
+		task ("When type 'vanilla.counter' in HTML Element Name field and attempt to save the remote app") {
+			RemoteApps.addApp(
+				entryName = "RemoteApp Test",
+				entryURL = "$testurl",
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase:722]: syntax ${} and camel case
${testURL}

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase (hunk 1 above)
Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase]: Hey John in this case and to fill in a url field. Can it have letters in uppercase?

**Comment by mateusralv** [comment]
ci:test:frontend-remote-apps

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase (hunk 1 above)
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase]: yes, can update to ${testURL}. I'm just matching what's existing in other tests ie: mainURL

Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase]: Ok, fixed! Thanks John 😃

**Comment by john-co** [comment]
cherry picked test fixes to https://github.com/liferay-frontend/liferay-portal/pull/2594

**Pull request closed without merging**